qa_crc16_async_bb is flaky

The `qa_crc16_async_bb` test added in #5700 has already failed three times in CI:

* https://github.com/gnuradio/gnuradio/runs/6312845070?check_suite_focus=true
* https://github.com/gnuradio/gnuradio/runs/6315190144?check_suite_focus=true
* https://github.com/gnuradio/gnuradio/runs/6316453166?check_suite_focus=true

```
 FAIL: test_crc16 (__main__.qa_crc16_async_bb)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "$SRC_DIR/gr-digital/python/digital/qa_crc16_async_bb.py", line 48, in test_crc16
    self.assertEqual(dbg_check.num_messages(), 1)
AssertionError: 0 != 1
```

The failure looks similar to the one corrected in #5754, where messages don't make it through before the flow graph shuts down.

So far I haven't been able to reproduce a failure locally. If the test continues to be problematic, it may be best to remove it until the source of the problem is tracked down.

--- a/gr-digital/lib/crc16_async_bb_impl.cc
+++ b/gr-digital/lib/crc16_async_bb_impl.cc
@@ -77,7 +77,7 @@ void crc16_async_bb_impl::check(pmt::pmt_t msg)
     const uint8_t* bytes_in = pmt::u8vector_elements(bytes, pkt_len);
 
     crc = d_crc_ccitt_impl.compute(bytes_in, pkt_len - 2);
-    if (crc != *(unsigned int*)(bytes_in + pkt_len - 2)) { // Drop package
+    if (memcmp(&crc, bytes_in + pkt_len - 2, 2)) { // Drop package
         d_nfail++;
         return;
     }

--- a/gr-digital/lib/crc16_async_bb_impl.cc
+++ b/gr-digital/lib/crc16_async_bb_impl.cc
@@ -55,8 +55,7 @@ void crc16_async_bb_impl::calc(pmt::pmt_t msg)
     const uint8_t* bytes_in = pmt::u8vector_elements(bytes, pkt_len);
     std::vector<uint8_t> bytes_out(2 + pkt_len);
 
-    crc = process_crc(bytes_in, pkt_len);
-
+    crc = d_crc_ccitt_impl.compute(bytes_in, pkt_len);
     memcpy((void*)bytes_out.data(), (const void*)bytes_in, pkt_len);
     memcpy((void*)(bytes_out.data() + pkt_len), &crc, 2);
 
@@ -65,16 +64,6 @@ void crc16_async_bb_impl::calc(pmt::pmt_t msg)
         bytes_out.data()); // this copies the values from bytes_out into the u8vector
     pmt::pmt_t msg_pair = pmt::cons(meta, output);
     message_port_pub(d_out_port, msg_pair);
-}
-
-unsigned int crc16_async_bb_impl::process_crc(const uint8_t* bytes_in,
-                                              size_t n_bytes_prcss)
-{
-
-    unsigned int result;
-
-    result = d_crc_ccitt_impl.compute(bytes_in, n_bytes_prcss);
-    return result;
 }
 
 void crc16_async_bb_impl::check(pmt::pmt_t msg)
@@ -87,8 +76,7 @@ void crc16_async_bb_impl::check(pmt::pmt_t msg)
     size_t pkt_len(0);
     const uint8_t* bytes_in = pmt::u8vector_elements(bytes, pkt_len);
 
-    crc = process_crc(bytes_in, pkt_len - 2);
-
+    crc = d_crc_ccitt_impl.compute(bytes_in, pkt_len - 2);
     if (crc != *(unsigned int*)(bytes_in + pkt_len - 2)) { // Drop package
         d_nfail++;
         return;

--- a/gr-digital/lib/crc16_async_bb_impl.h
+++ b/gr-digital/lib/crc16_async_bb_impl.h
@@ -25,7 +25,6 @@ private:
     pmt::pmt_t d_in_port;
     pmt::pmt_t d_out_port;
 
-    unsigned int process_crc(const uint8_t* bytes_in, size_t n_bytes_prcss);
     void calc(pmt::pmt_t msg);
     void check(pmt::pmt_t msg);
 

--- a/gr-digital/lib/crc32_async_bb_impl.cc
+++ b/gr-digital/lib/crc32_async_bb_impl.cc
@@ -79,7 +79,7 @@ void crc32_async_bb_impl::check(pmt::pmt_t msg)
     const uint8_t* bytes_in = pmt::u8vector_elements(bytes, pkt_len);
 
     crc = d_crc_impl.compute(bytes_in, pkt_len - 4);
-    if (crc != *(unsigned int*)(bytes_in + pkt_len - 4)) { // Drop package
+    if (memcmp(&crc, bytes_in + pkt_len - 4, 4)) { // Drop package
         d_nfail++;
         return;
     }

